packages/forms/resources/views/components/file-upload.blade.php

```diff
@@ -35,7 +35,7 @@
             uploadProgressIndicatorPosition: '{{ $getUploadProgressIndicatorPosition() }}',
             uploadUsing: async (fileKey, file, success, error, progress) => {
                 $wire.upload(`{{ $getStatePath() }}.${fileKey}`, file, () => {
-                    success()
+                    success(fileKey)
```

Comment:
Is it really necessary to pass `fileKey` from the component when we don't modify it?

Reply:
It's not necessary in our implementation, but if someone has a custom view or JS then this could be useful to them. It doesn't hurt to pass the key anyway :)